tokio/src/net/driver/readiness.rs

```diff
@@ -0,0 +1,299 @@
+use std::ops;
+
+const READABLE: usize = 0b0_01;
+const WRITABLE: usize = 0b0_10;
+const READ_CLOSED: usize = 0b0_0100;
+const WRITE_CLOSED: usize = 0b0_1000;
```

Comment:
This is missing AIO (not supported), LIO (not supported), error and priority. This is the full list (from https://github.com/tokio-rs/mio/pull/1145):

```rust
const READABLE: usize = 0b00_000_001;
const WRITABLE: usize = 0b00_000_010;
const AIO: usize = 0b00_000_100;
const LIO: usize = 0b00_001_000;
const ERROR: usize = 0b00_010_000;
const READ_CLOSED: usize = 0b00_100_000;
const WRITE_CLOSED: usize = 0b01_000_000;
const PRIORITY: usize = 0b10_000_000;
```

Reply:
It also just fits in 8 bits, if size is something to optimise for here.

Reply:
Yep I left those out for now. Readable and writeable readiness were supported before, and now Mio offers the cross-platform closed readiness that allows this change to remove platform-specific behavior. If support for these platform specific events provides values I think it should be done in a PR that is focused on adding functionality and not just updating.